code/controllers/subsystems/trade.dm

```diff
@@ -174,38 +175,72 @@ SUBSYSTEM_DEF(trade)
 
 	station.generate_offer()
 
+/datum/controller/subsystem/trade/proc/collect_counts_from(list/m)
```

Comment:
Comment what this function does, as your variable names are nondescript.

Reply:
Done